swarm/shed/index.go

```diff
@@ -37,15 +37,12 @@ type IndexItem struct {
 	Data            []byte
 	AccessTimestamp int64
 	StoreTimestamp  int64
-	// UseMockStore is a pointer to identify
```

Comment:
No, this field was introduced as the alternative to the `mock_index` is using only the `encode/decode` functions of the index to use mock store if a flag is set, however, we want the modes to control if mock is to be used, so we need to indicate it on the IndexItem.

Reply:
Yes correct, I reverted the removal.